components/object-property-list/object-property-list-item-tooltip-help.js

```diff
@@ -8,6 +8,7 @@ import { ObjectPropertyListItem } from './object-property-list-item.js';
  * A single object property, to be used within an object-property-list,
  * rendered as a help tooltip and with an optional icon.
  * @slot - Default content placed inside of the tooltip
+ * @slot icon - Optional slot for a custom icon
```

Comment:
I believe these slots should also be added to the hidden content section under `Properties` for object-property-list-tooltip-help.

Should the icon slot also be demoed?